goose3/cleaners.py

```diff
@@ -162,10 +162,10 @@ def clean_para_spans(self, doc):
             self.parser.drop_tag(item)
         return doc
 
-    def get_flushed_buffer(self, replacement_text, doc):
+    def get_flushed_buffer(self, replacement_text):
         return self.parser.textToPara(replacement_text)
 
-    def get_replacement_nodes(self, doc, div):
```

Comment:
could these break users current usages?

Reply:
I guess it could be possible that someone is calling this directly, but it is not documented on read-the-docs so I wouldn't expect it to be used; I believe that anything that is not in the online documentation should not be believed to be unchanging.